docs/Server.md

```diff
@@ -131,7 +131,7 @@ const serverFactory = (handler, opts) => {
   return server
 }
 
-const fastify = Fastify({ serverFactory })
+const fastify = Fastify({ serverFactory, modifyCoreObjects: false })
```

Comment:
Can you please add a comment explaining why we are passing the option?

Reply:
You can find it a few lines after :)